### PR TITLE
Update schedule page designs

### PR DESCRIPTION
**Pull request by aldenlamp**
Updated schedule page designs to match the Figma (Shown Below).

On iPhone 12 Pro Max:
iPhone 12 Pro Max

On iPhone SE - 2nd generation:
iPhone SE - 2nd generation

### CODE REVIEW

**Comment by PatrickKan** [comment]
@jeffkim2001 can you add some more test events to the api to test out the event list more

**Comment by jeffkim2001** [comment]
Changes that should be made:

1. In the bottom left corner of each event cell, there is a blue Join Stream! button. Change "Join Stream!" to "X Points!", where X can be a hard coded number. Convert the blue view (which should now be a points view) from a button to a HIView with the same colors and shape so that it is not clickable.
2. Some of the event titles are cut off, which is not what we want. Title labels should allow however many lines are necessary for the full event title to be presented in each event cell. Try setting numberOfLines = 0 for the title label.
3. The segmented control on the top of the schedule page should have the actual dates of HackIllinois 2021. There should be four tabs, which are labeled (from left to right): Friday 09, Saturday 10, Sunday 11, Monday 12.
4. In the bottom right corner of each event cell, there is a tag indicating the event type. For each event, present the event type that is currently stored in the HackIllinois API and color code it. Tags of the same event type should be printed in the same color and tags of different event types should be printed in different colors.

**Comment by PatrickKan** [comment]
Another thing, the times are being pulled from https://api.hackillinois.org/upload/blobstore/times/ right now, so we'd need to add a Monday time, or at least hardcode it for now. Times are store in epoch time by the way, a converter to local time is here https://www.epochconverter.com/

File: HackIllinois/ViewControllers/HIEventListViewController.swift [hunk 1]
```diff
@@ -99,19 +102,19 @@ extension HIEventListViewController: HIEventCellDelegate {
...
+//        }
+//        previewingContext.sourceRect = tableView.rectForRow(at: indexPath)
+//        eventDetailViewController.event = event
+//        return eventDetailViewController
+//    }
+//
+//    func previewingContext(_ previewingContext: UIViewControllerPreviewing, commit viewControllerToCommit: UIViewController) {
+//        self.present(viewControllerToCommit, animated: true, completion: nil)
+//    }
+//}
```
Comment (PatrickKan) [inline HackIllinois/ViewControllers/HIEventListViewController.swift:120]: Add a note of what this used to do?

**Review by PatrickKan — APPROVED** [review]

File: HackIllinois/ViewControllers/HIEventListViewController.swift (hunk 1 above)
Comment (jeffkim2001) [reply HackIllinois/ViewControllers/HIEventListViewController.swift:120]: The note stating that event detail view controller will not be used is at the top of the HIEventListViewController.swift file.

**Pull request merged**